kuma/users/tests/test_templates.py

```diff
@@ -22,6 +22,23 @@
 from ..models import User, UserBan
 
 
+def test_signin_landing(db, client, settings):
+    settings.MULTI_AUTH_ENABLED = True
+    response = client.get(reverse('socialaccount_signin'))
+    github_login_url = '/users/github/login/'
+    google_login_url = '/users/google/login/'
+    # first, make sure that the page loads
+    assert response.status_code == 200
+    doc = pq(response.content)
+    # ensure the title is accurate
+    assert 'Create your MDN Web Docs Account' in doc.find('h1').text()
```

Comment:
The title is not dynamic based on some business logic. I think this assertion should be avoided because generally text should be avoid to be tested unless the test is around some important phrasing that might or might not show up.

Reply:
Yeah, I was not sold on doing this but, I saw some other tests were testing similar scenarios so I through it in there.